tests/Unit/Evolution/DgSubcell/Test_CartesianFluxDivergence.cpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Evolution/DgSubcell/CartesianFluxDivergence.hpp"
+#include "Evolution/DgSubcell/Mesh.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeArray.hpp"
+
+namespace {
+template <size_t Dim>
+void test() {
+  const size_t num_pts_1d = 5;
+  const Index<Dim> subcell_extents{num_pts_1d};
+  for (size_t d = 0; d < Dim; ++d) {
+    CAPTURE(d);
+    auto extents = make_array<Dim>(num_pts_1d);
+    ++gsl::at(extents, d);
+    const auto basis = make_array<Dim>(Spectral::Basis::FiniteDifference);
+    auto quadrature = make_array<Dim>(Spectral::Quadrature::CellCentered);
+    gsl::at(quadrature, d) = Spectral::Quadrature::FaceCentered;
+    const Mesh<Dim> subcell_face_mesh{extents, basis, quadrature};
+
+    DataVector dt_var{subcell_extents.product(), 0.0};
```

Comment:
can you make this non-zero to test that `add_cartesian_flux_divergence` adds instead of sets